tools/lib/capitalization.py

```diff
@@ -138,6 +138,7 @@
     r"your-organization",
     # Used in invite modal
     r"or",
+    r"Welcome Bot",
```

Comment:
This should go next to "Notification Bot" and maybe now have a shared comment that these are system bot names.

Reply:
``` diff
diff --git a/tools/lib/capitalization.py b/tools/lib/capitalization.py
index c11116afb7..950765b00b 100644
--- a/tools/lib/capitalization.py
+++ b/tools/lib/capitalization.py
@@ -138,7 +138,6 @@ IGNORED_PHRASES = [
     r"your-organization",
     # Used in invite modal
     r"or",
-    r"Welcome Bot",
     # Units
     r"MB",
     r"GB",
@@ -161,6 +160,8 @@ IGNORED_PHRASES = [
     r"to add a new line",
     # Used in showing Notification Bot read receipts message
     "Notification Bot",
+    # Used in strings around welcome bot custom messages
+    r"Welcome Bot",
     # Used in presence_enabled setting label
     r"invisible mode off",
     # Typeahead suggestions for "Pronouns" custom field type.
```